components/common/src/util/path.rs

```diff
@@ -113,17 +122,24 @@ pub fn interpreter_paths() -> Result<Vec<PathBuf>> {
                                 return Err(Error::FileNotFound(path));
                             }
                         },
-                        // Well, we're not running out of a package, there is no interpreter package
-                        // installed, it's not on `PATH`, what more can we do. Time to give up the
-                        // chase. Too bad, we were really trying to be helpful here.
                         None => {
-                            println!(
-                                "A interpreter installation is required but could not be \
-                                 found. Please install '{}' or put the \
-                                 interpreter's command on your $PATH. Aborting...",
-                                INTERPRETER_IDENT
-                            );
-                            return Err(Error::PackageNotFound(ident.to_string()));
+                            install::start(
+                                &mut ui::UI::default_with_env(),
+                                &default_bldr_url(),
+                                Some(&channel::STABLE_CHANNEL.to_string()),
+                                &(ident.clone(), *PackageTarget::active_target()).into(),
+                                &*PROGRAM_NAME,
+                                VERSION,
+                                FS_ROOT_PATH.as_path(),
+                                &cache_artifact_path(None::<String>),
```

Comment:
```suggestion
                                cache_artifact_path(None::<String>),
```
[`cache_artifact_path`](https://github.com/habitat-sh/core/blob/master/components/core/src/fs.rs#L147) returns `PathBuf`, which satisfies the `AsRef<Path>` requirement on the parameter.